pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -140,6 +141,7 @@ func (a *Actuator) updateMachineStatus(machine *clusterv1.Machine, awsStatus *pr
 
 	if !equality.Semantic.DeepEqual(machine.Status, machineCopy.Status) {
 		mLog.Info("machine status has changed, updating")
```

Comment:
I know this is how it currently is, but this line "as is" seems pointless; it doesn't tell me anything.